product-updates/security-manager-3-2-release-note.md

```diff
@@ -0,0 +1,21 @@
+---
+title: SysKit Security Manager 3.2.0 - Release Note
+author: Petra Filipi 
+date: 28/08/2018 
+description: This article describes a bug fix and improvement delivered in SysKit Security Manager 3.2.0
+---
+
+This is a service release containing a bug fix and minor improvement.
+
+[Try SysKit Security Manager Free](https://www.syskit.com/products/security-manager/download/)
+
+__Product version:__ 3.2.0  
+__Build number:__ 873      
+__Release date:__ August 28, 2018  
+
+## Improvement 
+We improved reports to show the full path information to each object (Site Relative URL, Library Relative URL etc.)
```

Comment:
Mozda bolje "All of our reports now show the full path information to each object" jer cudno zvuci we improved reports.
 
I fali zarez prije etc. :)